libraries/python/semantic-workbench-assistant/semantic_workbench_assistant/start.py

```diff
@@ -92,7 +92,7 @@ def main():
         logger.info("Enabling auto-reload ...")
 
     settings.host = args.host
-    settings.port = args.port or find_free_port(settings.host)
+    settings.port = args.port or int(os.getenv("PORT") or find_free_port(settings.host))
```

Comment:
the settings.port is already designed to read an env var of `ASSISTANT__PORT` (in upper or lower case). to support `PORT` as well, i recommend updating `settings.py`, line 32

```python
    port: int = Field(validation_alias=AliasChoices("PORT", "ASSISTANT__PORT"), default=0)
```

and reverting the change to this file